config/rbac/role.yaml

```diff
@@ -51,6 +51,8 @@ rules:
   resources:
   - serviceaccounts
   verbs:
+  - create
```

Comment:
Why do we need to create and delete serviceaccounts?

Reply:
I automatically added by the "kubebuilder" command, we don't actually need this, you're right.

Reply:
done 216d573.